### PR TITLE
Fixed errors on the event details screen

### PR DESCRIPTION
**Pull request by Mikael-UofA**
Made it so you can no longer view the sign up button if you're not able to sign up and hid the view attendees button if you're not the organizer

### CODE REVIEW

File: code/app/src/main/java/com/example/qrazyqrsrus/EventDetailsFragment.java [hunk 1]
```diff
@@ -87,13 +88,14 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         ImageView checkInQRView = rootView.findViewById(R.id.check_in_qr_view);
 
         //try to get event and attendee from bundle
-        //if attendee is not there, thats fine, if event not there, very bad.
+        //if attendee is not there, that's fine, if event not there, very bad.
```
Comment (john00003) [inline code/app/src/main/java/com/example/qrazyqrsrus/EventDetailsFragment.java:91]: you added an apostrophe to my comment ❤️

File: code/app/src/main/java/com/example/qrazyqrsrus/QRCodeScanHandler.java [hunk 2]
```diff
@@ -108,8 +108,19 @@ public void onResult(Event matchingEvent) {
...
-                                            //otherwise, they should be brought to the event details screen, but shown a dialog saying they cannot check-in without signing up first
-                                            callback.onNoResult(matchingEvent, 5);
+                                            FirebaseDB.userCheckedIntoEvent(user, matchingEvent, new FirebaseDB.UniqueCheckCallBack() {
+                                                @Override
+                                                public void onResult(boolean isUnique) {
+                                                    if (isUnique) {
+                                                        callback.onCheckInResult(matchingEvent);
+                                                    }
+                                                    else {
+                                                        //otherwise, they should be brought to the event details screen, but shown a dialog saying they cannot check-in without signing up first
```
Comment (john00003) [inline code/app/src/main/java/com/example/qrazyqrsrus/QRCodeScanHandler.java:118]: v ery nice

**Review by john00003 — APPROVED** [review]
all looks lovely, thanks Mikael!

**Review by Arber-Shala — APPROVED** [review]
👍

**Pull request merged**